javav2/example_code/pinpoint/Readme.md

```diff
@@ -2,30 +2,68 @@
 
```

Comment:
Update line 1:
# Amazon Pinpoint code examples for the SDK for Java

Reply:
fixed